utils/Dockerfile.llvm

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:12.04
+
+# Used to build llvm and sub-packages
+# See https://github.com/mapbox/mason/blob/master/scripts/llvm/base/README.md for details
+# docker build -t mason-llvm -f Dockerfile .
+# docker run -it mason-llvm bash
+
+# we mimic the travis path to ensure any hardcoded paths in deps
+# do not cause problem's when building (an example is freetype-config)
+ENV WORKINGDIR /home/travis/build/mapbox/mason/
+WORKDIR ${WORKINGDIR}
+
+COPY mason mason
+COPY mason.sh mason.sh
+COPY scripts scripts
+COPY utils utils
+
+RUN apt-get update -y && \
+ apt-get install -y vim python build-essential bash curl git-core ca-certificates software-properties-common python-software-properties --no-install-recommends
+
+RUN add-apt-repository -y ppa:ubuntu-toolchain-r/test && \
```

Comment:
Nice catch! This is only needed for `libstdc++6`: an updated version of that is needed at runtime by `cmake`. It is not a compile/link dependency of llvm, so I've added comments about this to the Dockerfile to make this clear: 1c73eb6.